TrafficCapture/captureKafkaOffloader/build.gradle

```diff
@@ -1,5 +1,6 @@
 
 plugins {
+    id 'java'
```

Comment:
Let's remove these java plugin references they are implied by importing the common conventions.